docs/en/observability/apm-ui/index.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[xpack-apm]]
+= Navigate the APM UI
```

Comment:
This section is kind of a beast, but I'm not really sure how I would organize it differently. Seems like you have your top 3 in the Kibana UI with Services, Traces, Dependencies, but most of the headings are service-specific? I wonder if it would make sense to organize the things that are service specific under the services heading to make the initial list under Navigate the APM UI less daunting?

Reply:
As discussed in our weekly sync, we're just going to "lift and shift" the content from the Kibana guide to here. Then we will revisit the larger information architecture of the APM section in a follow-up PR.